gcpcli.js

```diff
@@ -0,0 +1,53 @@
+const _ = require("lodash");
+const util = require("util");
+const childProcess = require("child_process");
+
+const exec = util.promisify(childProcess.exec);
+
+// eslint-disable-next-line no-multi-str
+const GCP_AUTH_COMMAND = "\
+export AUTH_FILE=$(mktemp) && \
```

Comment:
The point of creating a command in a single command is to avoid volume creation to pass the credentials file?